test/install_inf_convertor_test.rb

```diff
@@ -283,4 +283,16 @@
         .to eql "LLADDR='#{HWADDR}'"
     end
   end
+
+  describe "use_biosdevname?" do
```

Comment:
`describe "AllowUdevModify" do`, or really rename the method (with inverted logic, as the name suggests)